File: beakjoon/10989.js
const fs = require('fs'); 
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
const n = parseInt(input[0]);
const arr =[];
for(let i=1; i<=n; i++){
    arr= arr.push(parseInt(input[i]));
}
console.log(arr.sort((a,b)=>a-b).join('\n'));

File: beakjoon/10989.py
import sys
n = int(sys.stdin.readline())
arr=[] #list라는 리스트명은 사용 금지
for i in range(n):
    nums = int(sys.stdin.readline())
    arr.append(nums) #arr리스트에 nums값을 추가
arr.sort() #오름차순
for i in arr:
    print(i) 
